pulsar-broker/src/main/java/org/apache/pulsar/common/naming/NamespaceBundleFactory.java

```diff
@@ -242,11 +242,11 @@ public NamespaceBundle getBundleWithHighestThroughput(NamespaceName nsName) {
         if (loadManager instanceof ModularLoadManagerWrapper) {
             NamespaceBundles bundles = getBundles(nsName);
             double maxMsgThroughput = -1;
-            NamespaceBundle bundleWithHighestThroughput = null;
+            NamespaceBundle bundleWithHighestThroughput = bundles.getBundles().get(0);
```

Comment:
I don't understand, I think even if the bundle is updated, we should always be able to find the one with the highest load?

Reply:
If the bundle don't have any connected topic, the check `bundleData.getTopics() > 0` will never pass, so the `bundleWithHighestThroughput ` is null.